src/app/containers/ATIAnalytics/beacon/index.js

```diff
@@ -0,0 +1,27 @@
+import sendBeacon from '../../../lib/analyticsUtils/sendBeacon';
+import { buildATIEventTrackUrl } from '../atiUrl';
+
+export const sendEventBeacon = async ({
+  element,
+  component,
+  type,
+  label,
+  ...params
+}) => {
+  const preAtiEventUrl = buildATIEventTrackUrl({
+    ...params,
+    element,
+    component,
+  });
+  const atiEventUrl = buildATIEventTrackUrl({
+    ...params,
+    element,
+    component,
+    type,
+    label,
+  });
+
+  await Promise.all([sendBeacon(preAtiEventUrl), sendBeacon(atiEventUrl)]);
```

Comment:
Not sure of the need for 2 beacons? could do with some text in the PR description to explain why we need two :)